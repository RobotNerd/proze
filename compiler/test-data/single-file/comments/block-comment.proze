Title: Block comments

###

Chapter: Not applicable

This text will be ignored. None of it will be in the compiled text.

###

This line will be in the text.
The first part of this line will be in the text, ### this won't
neither with this
or this, ### and so will this.


Present ### not present ### present ### not present###not present### not present.
