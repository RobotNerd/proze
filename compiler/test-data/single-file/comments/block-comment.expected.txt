Block comments


This line will be in the text. The first part of this line will be in the text, and so will this.

Present present present present.
